changelogs/fragments/no_log_sanity_fixes.yaml

```diff
@@ -0,0 +1,3 @@
+---
+trivial:
```

Comment:
Minor comment. This should be `security_fixes` IMO. For example https://github.com/ansible-collections/arista.eos/pull/198/files#diff-71b2212c15b4a6ed8d0642ad1bc99f2471d111bb5c172eb645d403ef79a88ebeR2

Reply:
yeah that sounds better! will update